netflix-assignment/src/app/(route)/home/_components/ContentRow.tsx

```diff
@@ -0,0 +1,47 @@
+"use client"
+
+import { Movie } from '@/types/movie';
+import { useContentStore } from '../../../../store/useContentStore';
+import { TvShow } from '@/types/tvshows';
+
+type Props = {
+  title: string;
+
+  content: Movie[] | TvShow[];
+
+};
+
+export default function ContentRow({ title, content }: Props) {
+
+
+  if (!content || content.length === 0) return null;
+  const getAltText = (item: Movie | TvShow) => {
+      if ('title' in item) return item.title;
+      if ('name' in item) return item.name;
+      return 'No Title';
+    };
+  const getPrefix = (item: Movie | TvShow) => {
+      if ('title' in item) return 'movie';
+      if ('name' in item) return 'tv';
+      return 'No';
+    }
```

Comment:
if문도 좋지만 삼항 연산자를 사용해서 더 간결하게도 쓸 수 있을 것 같습니당

```suggestion
  const getAltText = (item: Movie | TvShow) => item.title || item.name || 'No Title';
  const getPrefix = (item: Movie | TvShow) => (item.title ? 'movie' : item.name ? 'tv' : 'No');

```